nav.go

```diff
@@ -661,13 +661,13 @@ func (nav *nav) reload() error {
 	clear(nav.regCache)
 
 	curr, err := nav.currFile()
-	nav.getDirs(nav.currDir().path)
+	nav.updateDirs(nav.currDir().path)
 	if err == nil {
 		last := nav.dirs[len(nav.dirs)-1]
 		last.files = append(last.files, curr)
 	}
 
-	return nil
+	return err
```

Comment:
It seems to make more sense to pass on the error to the caller instead of silently swallowing it. Thus, this seems like an actual bug found by the linter.